services/src/main/scala/cromwell/pipeline/service/GitLabProjectVersioning.scala

```diff
@@ -91,7 +115,22 @@ class GitLabProjectVersioning(httpClient: HttpClient, config: GitLabConfig)
 
   override def getFileVersions(project: Project, path: Path)(
     implicit ec: ExecutionContext
-  ): AsyncResult[List[Version]] = ???
+  ): AsyncResult[Seq[Version]] =
+    for {
+      projectVersions <- getProjectVersions(project)
+      fileCommits <- getFileCommits(project, path)
+    } yield {
```

Comment:
```suggestion
    val projectVersionsF = getProjectVersions(project)
    val fileCommitsF = getFileCommits(project, path)
    for {
      projectVersions <- projectVersionsF
      fileCommits <- fileCommitsF
    } yield {
```
Looks like you have two independent asynchronous requests. This is not so important, but if you make both of them outside `for-comprehesion`, you'll get the benefit from it since they will be done in parallel.

Reply:
Change applied it